### ISSUE TITLE
function of Null model 

### ISSUE THREAD

**Issue by xhxlilium** [ChiLiubio/microeco#170]

**Comment by ChiLiubio** [comment]
Yes.  Drift is the 'Undominated' part in the concept diagram (|RCbray| < 0.95). Maybe it is not the actual drift. We just view it as the undominated approximately. So you can adjust the explanation as you want. I think the ecological inference can also vary with the context.

Chi
